meetings/2022-12-05.md

```diff
@@ -0,0 +1,91 @@
+# 2022-12-05 Solid Interoperability
+
+* Time: 15:00UTC
+* Link: 
+
+## Present
+
+* Wouter Termont
+* Jamie Fiedler
+* elf Pavlik
+* Justin Bingham
+* Ángel Araya
+
+### Regrets
+
+* Laurens Debackere
+
+## Agenda
+
+* [Agent-Specific Discovery](https://github.com/solid/data-interoperability-panel/issues/289)
+* Open PRs
+
+## Scribes
+
+* Pavlik
+* Justin
+
+## Minutes
+
+
+### Agent-Specific Discovery
+
+Issue: https://github.com/solid/data-interoperability-panel/issues/289
+Spec: https://woutermont.github.io/solid/agent-specific-discovery.html
+
+* WT: I worte it based on arguments I put forwrad. To better align with different approaches to interop it might be better to split different aspects in separate specs. This is my first attempt, taking most of the agent registration aspects.
```

Comment:
```suggestion
* WT: I wrote it based on arguments I put forward. To better align with different approaches to interop, it might be better to split different aspects into separate specs. This is my first attempt, taking most of the agent registration aspects.
```